tests/test_code_to_func.py

```diff
@@ -88,24 +86,3 @@ def test_local_file_codeout():
     assert path.isfile(out), "output not generated"
 
     fn.run(handler="training", params={"p1": 5})
-
-
-def test_nuclio_py():
```

Comment:
I can't find this test and `test_nuclio_golang` in the files. Did you remove them for some reason or they should have been moved?

Reply:
renamed to `test_compiled_function_config_nuclio_python` and `test_compiled_function_config_nuclio_golang` and moved to `tests/api/crud/runtimes/nuclio/test_helpers.py`